crates/sel4-capdl-initializer/types/src/spec.rs

```diff
@@ -76,6 +76,8 @@ pub enum Object<'a, D, M> {
     PageTable(object::PageTable<'a>),
     AsidPool(object::AsidPool),
     ArmIrq(object::ArmIrq<'a>),
+    IrqMsi(object::IrqMsi<'a>),
+    IrqIoapic(object::IrqIoapic<'a>),
```

Comment:
Could you change the capitalization of a few identifiers, for the sake of consistency with the rest of the project?

`IrqIoapic` -> `IrqIOApic`
`IrqMSIHandler` -> `IrqMsiHandler`
`IrqIOAPICHandler` -> `IrqIOApicHandler`
`IrqMSIExtraInfo` -> IrqMsiExtraInfo`
`IrqIOAPICExtraInfo` -> `IrqIOApicExtraInfo`

Reply:
I think it's consistent now.